Updated Azure Container setup documentation

slancerk opened the pull request:
simply added `redis env's`  and `pgsslmode`, also changed `Command override-> Arguments override`

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/ToolJet/ToolJet?pullRequest=11346) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/ToolJet/ToolJet?pullRequest=11346) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/ToolJet/ToolJet?pullRequest=11346) it.</sub>

slancerk commented:
updated `docs/versioned_docs/version-3.0.0-LTS/setup/azure-container.md`
changed: `Command override` -> `Arguments override`
added:
```
   For redis connection ensure below environment variables are added:
   - **REDIS_HOST**
   - **REDIS_PORT**
   - **REDIS_USER**
   If using Azure Database for Postgresql -Flexible server, add:
   - **PGSSLMODE = require**
```
